requirements.txt

```diff
@@ -1,36 +1,26 @@
-altgraph==0.17
-amply==0.1.4
-cycler==0.10.0
-decorator==4.4.2
-docutils==0.17.1
-future==0.18.2
+Pillow==9.3.0
+PuLP==2.7.0
+PyQt5==5.15.7
+PyQt5-Qt5==5.15.2
+PyQt5-sip==12.11.0
+PyQt5-stubs==5.15.6.0
+XlsxWriter==3.0.3
+contourpy==1.0.6
+cycler==0.11.0
+fonttools==4.38.0
 grandalf==0.7
 grinpy==19.7a0
-importlib-metadata==4.11.3
-kiwisolver==1.4.2
-macholib==1.14
-matplotlib==3.4.2
-modulegraph==0.18
-netgraph==4.4.1
+kiwisolver==1.4.4
+matplotlib==3.6.2
+netgraph==4.10.2
```

Comment:
This dependency is showing a deprecation error:
```shell
DEPRECATION: netgraph is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
```

I think it is worth it to open a issue at Netgraphs repository to let them know we are having this problem.

Reply:
[Issue open](https://github.com/paulbrodersen/netgraph/issues/54)